src/shared/sentry/sentryadapter.cpp

```diff
@@ -70,6 +70,7 @@ void SentryAdapter::init() {
           &SentryAdapter::onLoglineAdded);
 
   sentry_options_t* options = sentry_options_new();
+  sentry_options_set_max_breadcrumbs(options, 500);
```

Comment:
Looks like the default is 100. Is that too small? Just curious why this is being changed. Also, consider adding a comment explaning why the default is being changed.